setupext.py

```diff
@@ -318,6 +324,8 @@ def setup_extension(self, ext, package, default_include_dirs=[],
                     dir = os.path.join(base, include)
                     if os.path.exists(dir):
                         ext.include_dirs.append(dir)
+                ext.include_dirs.extend(
+                        os.environ.get('CPLUS_INCLUDE_PATH', '').split(os.pathsep))
```

Comment:
I used it as a way to influence the base dirs without editing (=patching during build) setup.cfg and wondered why it didn't work...